src/js/eventInTimeAware.js

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+/* global fluid, floe */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // A grade that deals in time,
+    // and things related to time
+    fluid.defaults("floe.dashboard.eventInTimeAware", {
+        gradeNames: "fluid.modelComponent",
+        model: {
+            // A series of key-timestamp pairs
+            timeEvents: {
+                // created: "",
+                // lastModified: ""
+            },
+            formattedDates: {
+                "created": null,
```

Comment:
They are in use within the preferences framework as part of the so-called "primary schema" - e.g. https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/js/StarterSchemas.js#L248 - you can consult @cindyli and @jobara for help on this